File: marvelHeroes/src/main/java/com/futureworkshops/marvelheroes/presentation/character/favorite/FavoriteCharactersFragment.java
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroes.presentation.character.favorite;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.futureworkshops.marvelheroes.R;

import butterknife.ButterKnife;

/**
 * A simple {@link Fragment} subclass.
 * Use the {@link FavoriteCharactersFragment#newInstance} factory method to
 * create an instance of this fragment.
 */
public class FavoriteCharactersFragment extends Fragment {
    
    
    public static FavoriteCharactersFragment newInstance() {
        
        return new FavoriteCharactersFragment();
    }
    
    public FavoriteCharactersFragment() {
        // Required empty public constructor
    }
    
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
    
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        final View view = inflater.inflate(R.layout.fragment_favorite_characters, container, false);
       
        ButterKnife.bind(this, view);
        
        return view;
    }
    
}

File: marvelheroeskotlin/src/main/java/com/futureworkshops/marvelheroeskotlin/presentation/FavoriteCharactersFragment.kt
/*
 * Copyright (c) 2018 FutureWorkshops. All rights reserved.
 */

package com.futureworkshops.marvelheroeskotlin.presentation

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.futureworkshops.marvelheroeskotlin.R

/**
 * Created by dimitrios on 20/06/2018.
 * Copyright © 2018 - Future Workshops.
 * All rights reserved.
 */

class FavoriteCharactersFragment : Fragment() {
    
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_favorite_characters, container, false)
    }
    
    companion object {
        
        fun newInstance(): FavoriteCharactersFragment {
            return FavoriteCharactersFragment()
        }
    }
    
}